_docs/crowfacts.md

```diff
@@ -39,10 +67,34 @@ More information coming soon!
 More information coming soon!
 
 ### DynamoDB
+The information presented on the website was stored on a DynamoDB database table. This is a NoSQL database, and there are important factors to consider when implementing this in a project.
 
-More information coming soon!
+#### Consider your usage
+NoSQL databases do not require relational information, as would be the use-case for SQL databases. They are flexible, and columns can be introduced and removed as needed. If data integrity is necessary, than consider AWS's relational database options, such as [Amazon RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html).
```

Comment:
I reworded this section for accuracy and flow. If you disagree with any of these changes, feel free to ask me why I made them.

```suggestion
NoSQL databases do not require a defined relational schema as SQL databases do. They are flexible, allowing attributes to be introduced as needed. If the relationships between data points are important, then consider AWS's relational database options such as [Amazon RDS](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html).
```